src/vite-env.d.ts

```diff
@@ -24,6 +24,10 @@ declare module '@agoric/ertp' {
     COPY_SET: 'copySet',
     COPY_BAG: 'copyBag',
   };
+  export type Ratio = {
```

Comment:
This isn't really an `@agoric/ertp` type. If it were it would be in `ertp.d.ts`. Please maintain the truth of line 18 and not add any types here.

It's actually from `contractSupport`. Of course to start that declaration you'll need to specify the other keys but it's pretty much a copy-paste of the symbols in the one import. Then just set them all to any for now.

I tried generating a types file like I did for `ertp.d.ts` but it's much less amenable.

Reply:
Ah thanks for pointing this out and giving that a try. Fixed.